contrib/configure-opt

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+#
+# Copyright (C) Mellanox Technologies Ltd. 2001-2014.  ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+#
+
+#
+# UCX build for maximal performance using specific CPU.
+# No extra debugging or profiling code.
+#
+
+basedir=$(cd $(dirname $0) && pwd)
+$basedir/configure-release \
```

Comment:
sec - what scripts distro will use ? shell we separate "release" and "optimized"